frontend/components/FeatureCard/FeatureCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box as ChakraBox, useColorModeValue } from "@chakra-ui/react";
+import { Center, Icon, Text } from "@chakra-ui/react";
+import React from "react";
+import { IconType } from "react-icons";
+
+export interface FeatureCardProps {
+  /**
+   * Box contents
+   */
+  title: string;
+  description: string;
+  icon: IconType;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const FeatureCard: React.FC<FeatureCardProps> = ({
+  title,
+  description,
+  icon,
+}: FeatureCardProps) => {
+  return (
+    <ChakraBox
+      boxShadow="5px 5px 0px 0px #4e93bb, -15px -20px 0px 0px #96cfee"
+      width="270px"
+      height="200px"
+      position="relative"
+      bg={useColorModeValue("white.light", "white.dark")}
```

Comment:
I noticed that our theme file doesn't contain any reference to light/dark variants of white, I was wondering where this value was referenced from?

Reply:
It's also showing blue as the box color on my side!